dev-lang/clips/clips-6.3.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="C Language Integrated Production System"
+DESCRIPTION="
+CLIPS is an OPS-like forward chaining production system \
+written in ANSI C by NASA. The CLIPS inference engine \
+includes truth maintenance, dynamic rule addition, and \
+customizable conflict resolution strategies."
+HOMEPAGE="http://www.clipsrules.net"
+COPYRIGHT="1985-2015, NASA, Gary Riley"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/clipsrules/files/CLIPS/6.30/clips_core_source_630.zip"
+CHECKSUM_SHA256="01555b257efae281199b82621ad5cc1106a395acc095b9ba66f40fe50fe3ef1c"
+SOURCE_DIR="clips_core_source_630"
+
+ARCHITECTURES="?x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
Suggestion as clips builds ok with gcc2:
```diff
-ARCHITECTURES="?x86_gcc2 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
```
You should also remove the entries for "secondaryArchSuffix" then